main-app/app/scripts/core/controllers/core-controller.js

```diff
@@ -2,9 +2,10 @@
     'use strict';
     angular.module('Tombola.Games.Bingo90.Core')
 
-        .controller('MainController', ['$scope','LoginApi','GameApi','LoginProxy',  function($scope, loginApi, gameApi, loginProxy) {
+        .controller('MainController', ['$scope','LoginApi','GameApi','LoginProxy', 'GameProxy',  function($scope, loginApi, gameApi, loginProxy, gameProxy) {
             $scope.gameTitle = 'tombola';
             $scope.model = loginProxy;
+            $scope.gameModel = gameProxy;
 
             $scope.login = function (){
                 loginApi.loginButton($scope.username, $scope.password);
```

Comment:
you shouldn't need to pump the values in like this: the loginProvider should expose a username/password values, and a login method.
It should also keep track of whether the user is logged in, and provide logout functionality. 
Done this way, the "login provider" is actually an "authenticated provider" - encapsulated and isolated